modules/connectivity/ConnectionQuality.js

```diff
@@ -256,17 +214,8 @@ export default class ConnectionQuality {
                 this._updateRemoteStats(participant.getId(), payload);
             });
 
-        // Listen to local statistics events originating from the RTC module
-        // and update the _localStats field.
-        // Oh, and by the way, the resolutions of all remote participants are
-        // also piggy-backed in these "local" statistics. It's obvious, really,
```

Comment:
Does the (passive-aggressive) comment no longer apply?

Reply:
The comment didn't make sense to me. The local stats are the stats we get from the pc and the remote stats are the ones we get from the remote endpoints through the bridge channel. I don't think we need to keep track of the remote resolution in `_remoteStats` since it is obtained from the local pc `getStats`, that will unnecessarily complicate the logic. WDYT ?

Reply:
SGTM!